docs/dckrman.sh

```diff
@@ -5,3 +5,5 @@ for i in $@; do
     echo .so man1/$b > $filename
 done
 echo .so man5/containerfile.5 > $(dirname $1)/dockerfile.5
+echo .so man5/containerignore.5 > $(dirname $1)/.dockerignore.5
```

Comment:
Installing a hidden man page on unix seems unusual. Other commands do not do this. gitignore given as example.

Reply:
It seems to be rare:
```console
$ find /usr/share/man -name '.*'
/usr/share/man/man1/..1.gz
/usr/share/man/man5/.k5identity.5.gz
/usr/share/man/man5/.k5login.5.gz
```
Running `man 5 <TAB>` I see many rc and conf files that are traditionally dotfiles. I think that makes dotless a pretty strong convention.

Reply:
Sure, but you did find a couple. And I find no reason not to do it, just because there is not a lot of precedence. Interesting for conversation, but I don't see how this hurts anyone and might actually help someone.

Reply:
No strong feelings here, just looked odd.